testing/ios/IosUnitTests/IosUnitTests.xcodeproj/xcshareddata/xcschemes/IosUnitTests.xcscheme

```diff
@@ -78,6 +78,12 @@
             ReferencedContainer = "container:IosUnitTests.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "--icu-data-file-path=Frameworks/Flutter.framework/icudtl.dat"
+            isEnabled = "YES">
```

Comment:
Is this needed?

Reply:
Yes. Without this the test will fail due to failure to load the ICU data file.

Reply:
Filed https://github.com/flutter/flutter/issues/108221

Reply:
I think this is fine - in other tests we use the command line argument to load the ICU data file (see run_tests.py).